openshift/templates/site-builder.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+#set -euo pipefail
+
+if [ ! -z "${OC_DEPLOY_PROJECT}" ] && [ ! -z "${APP_DOMAIN}" ] && [ ! -z "${SITE_NAME}" ] && [ ! -z "${POOL_NAME}" ]; then
+    echo >&2 "Project:   ${OC_DEPLOY_PROJECT}"
+    echo >&2 "App Name:  ${APP_DOMAIN}"
+    echo >&2 "Site Name: ${SITE_NAME}"
+    echo >&2 "Pool Name: ${POOL_NAME}"
+    
+    whoAmI="$(oc whoami 2> /dev/null)"
+    # This means i am logged in.
+    if [ ! -z "${whoAmI}" ]; then
+        templates="$(dirname $0)";
+        #apply=""
+        apply=" | oc apply -f -"
```

Comment:
Does this variable get used anywhere?

Reply:
actual it doesn't it had a purpose, but didn't work, removed it.. 
Thanks.